Dockerfile

```
@@ -13,6 +13,8 @@ RUN ./gradlew clean bootJar --no-daemon
 # Stage 2: Runtime
 FROM eclipse-temurin:21-jre
 
+RUN apt-get update && apt-get install -y --no-install-recommends curl && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)